Applications/spire/include/spire/ui/dropdown_menu.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef SPIRE_DROPDOWN_MENU_HPP
+#define SPIRE_DROPDOWN_MENU_HPP
+#include <QComboBox>
+#include "spire/ui/ui.hpp"
+
+namespace Spire {
+
+  //! Displays a Spire-styled QComboBox.
+  class DropdownMenu : public QWidget {
+    public:
+
+      //! Constructs a DropdownMenu with specified items. The first item
+      //! in the initializer_list is the default selected item.
+      /*
+        \param items The items to display in the menu.
+        \param parent The parent widget to the DropdownMenu.
+      */
+      DropdownMenu(const std::initializer_list<QString>& items,
```

Comment:
This constructor needs to be ```explicit```:

https://en.cppreference.com/w/cpp/language/explicit

Reply:
I understand that this prevents implicit conversion, but how is it a problem here?